src/components/find-block/find-block.styles.js

```diff
@@ -1,19 +1,34 @@
 export const styles = {
   container: {
     mt: { xs: '64px', md: '84px' },
+    gap: '20px',
+
+    '&:first-child > div > :last-of-type': {
```

Comment:
This selector looks a bit overwhelming.
Besides, I got an error in the console:
<img width="967" alt="image" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/110097862/cc3b6a3c-0aa9-4872-b4ee-40ef803f707a">

Maybe it's better to apply this changes directly in the `TitleBlock.styles.ts`?

Reply:
I agree. Fixed